auth-web/src/services/business.services.ts

```diff
@@ -93,4 +93,37 @@ export default class BusinessService {
       `${ConfigHelper.getLegalAPIV2Url()}/businesses/${businessIdentifier}/documents/summary`, config)
     CommonUtils.fileDownload(data?.data, `${businessIdentifier} Summary.pdf`, data?.headers['content-type'])
   }
+
+  static async fetchContinuationBusinesses (): Promise<any> {
+    // Mock data simulating the expected response structure from the API
+    return new Promise(resolve => resolve({
+      data: [
+        {
+          date: 'July 3, 2024',
+          nrNumber: 'NR 0001234',
+          businessIdentifier: 'LN958001',
+          completingParty: 'John Doe',
+          status: 'Awaiting Review'
+        },
+        {
+          date: 'April 4, 2024',
+          nrNumber: 'NR 0001235',
+          businessIdentifier: 'LN786002',
+          completingParty: 'Jane Smith',
+          status: 'Change Requested'
+        },
+        {
+          date: 'June 15, 2024',
+          nrNumber: 'NR 0001239',
+          businessIdentifier: 'LN965002',
+          completingParty: 'Test Mark',
+          status: 'Rejected'
+        }
+      ]
+    } as any))
+
+    // const url = `${ConfigHelper.getLegalAPIV2Url()}/businesses/continuation-in`
+    // const response = await axios.get(url)
+    // return response.data
+  }
```

Comment:
Mock some sample response here. The url is to be updated when "get reviews" endpoint is available

Reply:
Right. Waiting for bcgov/entity#21929.